File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F06.P2.go
package main

import "fmt"

func probe() int32 {
	// BEGIN PROBE F06.P2
	divmod2 := func(a, b int32) (int32, int32) { return a / b, a % b }
	q, r := divmod2(7, 3)
	return q + r
	// END PROBE F06.P2
}

func main() {
	fmt.Println(probe())
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/typescript/F06.P2.ts
function probe(): number {
// BEGIN PROBE F06.P2
const divmod2 = (a: number, b: number): [number, number] => [Math.trunc(a / b), a % b]
const [q, r] = divmod2(7, 3)
return q + r
// END PROBE F06.P2
}

console.log(probe())
